src/DeviceInterfaceValues.tsx

```diff
@@ -394,6 +410,15 @@ const SendInterfaceDataModal = ({
                   <option value="true">true</option>
                   <option value="false">false</option>
                 </Form.Select>
+              ) : selectedMapping.type === 'datetime' ? (
+                <DatePicker
+                  selected={selectedDate}
+                  onChange={(date: Date) => handleDateTimeChange(date)}
```

Comment:
considering how the `Date` is handled, i would suggest adding a " (UTC)" string after the inputbox so users are aware they are selecting a date/time in the UTC timezone